DataConnectors/Forcepoint CASB.json

```diff
@@ -0,0 +1,108 @@
+        },
+      {
+            "description" : "Top 5 Users by Number of Failed Attempts ",
+            "query": "CommonSecurityLog \n| extend outcome = split(split(AdditionalExtensions, \";\", 2)[0], \"=\", 1)[0]\n| extend reason = split(split(AdditionalExtensions, \";\", 3)[0], \"=\", 1)[0]\n| where outcome ==\"Failure\"\n| summarize Count= count() by DestinationUserName\n| render barchart"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (CEF)",
+            "lastDataReceivedQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CASB\"\n   | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CASB\"\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "1. Linux Syslog agent configuration",
+            "description": "Install and configure the Linux agent to collect your Common Event Format (CEF) Syslog messages and forward them to Azure Sentinel.\n\n> Notice that the data from all regions will be stored in the selected workspace. ",
+            "innerSteps": [
+                {
+                    "title": "1.1 Select or create a Linux machine",
+                    "description": "Select or create a Linux machine that Azure Sentinel will use as the proxy between your security solution and Azure Sentinel this machine can be on your on-prem environment, Azure or other clouds."
+                },
+                {
+                    "title": "1.2 Install the  Forcepoint CASB CEF collector on the Linux machine",
+                    "description": "To Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector collects CEF messages on port 514 TCP.\n\n>Steps to install and configure Forcepoint CASB connector are provided in the Forcepoint CASB integration installation guide which can be found in this link:\nhttps://support.forcepoint.com/knowledgebase#sort=relevancy\n\n",
```

Comment:
1. This link https://support.forcepoint.com/knowledgebase#sort=relevancy points to a general table of contents, rather than specific page to configure. Can you please update with an appropriate link that goes to the right page? 
2. Also have the link as a  redirection URL that you can manage so that if the actual link changes, it doesn’t need a data connector update for link fix and re-deployment which takes time. Putting a fixed link creates issues with changes to links requiring re-deployment of data connector which takes a lot of time. 
3. While updating with the link make the format as "[Forcepoint CASB integration installation guide](https://linkname)" so that you don't need the text 'which can be found in this link'...